pkg/util/command.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import (
+	"bytes"
+	"os/exec"
+)
+
+// ExecCommand executes a command using the VPN binary
+func ExecCommand(command string, args []string) (string, error) {
+	var out bytes.Buffer
+
+	cmd := exec.Command(command, args...)
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return "", err
+	}
```

Comment:
With this, you get more info if there is an error, the err outs from exec are almost always just `exit 1/2`, which isn't helpful.
```suggestion
	var out, errOut bytes.Buffer

	cmd := exec.Command(command, args...)
	cmd.Stdout = &out
	cmd.Stderr = &errOut
	err := cmd.Run()
	if err != nil {
		return errOut.String(), err
	}
```

Reply:
nice, thanks for that. The user of this function should then be aware that the interesting error is the string returned and not the error, right? I'll add a comment